initial version (#1062)
RevenueCat/docs: docs/dashboard-and-metrics/charts/paywall-ltv-chart.md

---
title: Paywall LTV Chart
slug: paywall-ltv-chart

> hidden: false

Reviewer: ```suggestion
hidden: true
```

---

> The Paywall LTV chart measures the realized lifetime value generated from customers who had an initial conversion within 3 days of their first paywall impression. It lets you view total revenue, revenue net of taxes, or proceeds, and also average LTV per viewer.

Reviewer: ```suggestion
The Paywall LTV chart measures the realized lifetime value generated from purchases that derived from an initial conversion within 3 days of a customer's first impression on a given paywall. It lets you view total revenue, revenue net of taxes, or proceeds, and also average LTV per viewer.
```
